src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1120,6 +1120,50 @@ input InputUnpreferred {
     useUnpreferredRoutesPenalty: Int
 }
 
+enum RoutingErrorCode {
+    """No transit connection was found between the origin and destination withing the operating day or the next day"""
+    noTransitConnection
```

Comment:
In the HSL API we generally use upper case notation for the enum values. Example: https://github.com/opentripplanner/OpenTripPlanner/blob/ee1a02f02ce238859181890cb6fd12ba86c820fc/src/ext/resources/legacygraphqlapi/schema.graphqls#L162-L198

(Personally I don't like that too much either, but lets keep it consistent)

Reply:
This is true! I read the schema of `routingErrors` field Transmodel API. We really have 2 competing conventions here and some refactoring will be inevitable when migrating to upcoming uniform GraphQL API.